### PR TITLE
Fix bug in ManualInterval when limits were manually set to zero

### PR DESCRIPTION
**Pull request by astrofrog**
The dangers of using 'a or b' when a is zero...

### CODE REVIEW

File: astropy/visualization/tests/test_interval.py [hunk 1]
```diff
@@ -30,6 +31,22 @@ def test_manual_defaults(self):
...
+        # Regression test for a bug that caused ManualInterval to compute the
+        # limit (min or max) if it was set to zero.
+        interval = ManualInterval(vmin=0, vmax=0)
+        vmin, vmax = interval.get_limits(self.data)
+        np.testing.assert_allclose(vmin, 0)
+        np.testing.assert_allclose(vmax, 0)
+
+    def test_manual_defaults_with_nan(self):
+        interval = ManualInterval()
+        data = self.data.copy()
```
Comment (larrybradley) [inline astropy/visualization/tests/test_interval.py]: This fails because `self.data` is a list:  -> `data = np.copy(self.data)`

Comment (astrofrog) [inline astropy/visualization/tests/test_interval.py]: Ah I missed this because I'm using Python 3 :)

Comment (larrybradley) [inline astropy/visualization/tests/test_interval.py]: It will be so nice to support only Python 3!  :)

**Comment by pllim** [comment]
Squash? 😄

**Pull request merged**